src/pages/edge/adobe-journey-optimizer/in-app-message/tutorials/javascript-from-native.md

```diff
@@ -16,37 +17,45 @@ import Tabs from './tabs/javascript-from-native.md'
 
 You can execute JavaScript in an in-app message from native code by completing the following steps:
 
-* [Implement and assign a `MessagingDelegate`](#implement-and-assign-a-messagingdelegate)
+* [Implement and assign a `PresentationDelegate`/ `MessagingDelegate`](#implement-and-assign-a-messagingdelegate)
 * [Obtain a reference to the web view](#obtain-a-reference-to-the-web-view)
 * [Call the JavaScript method](#call-the-javascript-method)
 
-## Implement and assign a `MessagingDelegate`
+## Implement and assign a `PresentationDelegate`/ `MessagingDelegate`
 
-To register a JavaScript event handler with a `Message` object, you will first need to implement and set a `MessagingDelegate`.
+To register a JavaScript event handler with a `Message` object, you will first need to implement and set a `PresentationDelegate` or `MessagingDelegate`, based on which SDK you are using.
 
-For more detailed instructions on implementing and using a MessagingDelegate, please read the [tutorial on using MessagingDelegate](./messaging-delegate.md).
+For more detailed instructions on implementing and using a PresentationDelegate or MessagingDelegate, please read the [tutorial on using MessagingDelegate](./messaging-delegate.md).
```

Comment:
recommend-  "_Please read the [tutorial](....) for detailed instructions on implementing and using PresentationDelegate/MessagingDelegate_" so that the delegate wording is not repeated.